homeassistant/components/sonarr/sensor.py

```diff
@@ -56,6 +60,10 @@ class SonarrSensorEntityDescription(
         native_unit_of_measurement=DATA_GIGABYTES,
         entity_registry_enabled_default=False,
         value_fn=lambda data: f"{sum(disk.freeSpace for disk in data) / 1024**3:.2f}",
+        attributes_fn=lambda data: {
+            d.path: f"{d.freeSpace / 1024**3:.2f}/{d.totalSpace / 1024**3:.2f}{DATA_GIGABYTES} ({(d.freeSpace / 1024**3) / (d.totalSpace / 1024**3) * 100:.2f}%)"
+            for d in data
+        },
```

Comment:
If the lambda gets too complicated, it's better to use a dedicated function instead.